README.md

```diff
@@ -1,7 +1,40 @@
-# nulecule-library
-A set of [nulecule](https://github.com/projectatomic/nulecule)-ized applications that can be used for reference or as part of your applications
+# Nulecule library
 
-###Communication channels
+A set of [nulecule](https://github.com/projectatomic/nulecule)-ized applications that can be used for reference or as part of your deployment.
```

Comment:
Given the difficulty of the term and because this library will hopefully be a direct reference point for some deployable apps at some point, I think a small one with more "in your face reference" to the getting started guide and the nulecule spec would be useful.